### PR TITLE
TA-test-39180

### PR DESCRIPTION
**Pull request by priyankaborawake**
Signed-off-by: priyanka.borawake <priyanka.borawake@seagate.com>

# Problem Statement
- TA-test-39180
- 
# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by priyankaborawake** [comment]
[TEST-39180_TestIOWorkload_20220323042513.log](https://github.com/Seagate/cortx-test/files/8332413/TEST-39180_TestIOWorkload_20220323042513.log)

File: tests/cft/test_io_workload.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    """Test suite for IO workloads."""
+
+    @classmethod
+    def setup_class(cls):
+        """Setup class."""
+        cls.prov_obj = ProvDeployK8sCortxLib()
+        cls.s3t_obj = S3TestLib()
+
+    @pytest.mark.lc
+    @pytest.mark.sanity
```
Comment (sonalk0209) [inline tests/cft/test_io_workload.py:38]: can we add marker for io path or io_workload whichever is used for data path

File: tests/cft/test_io_workload.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    @pytest.mark.tags("TEST-39180")
+    def test_39180(self):
+        """Basic IO test."""
+        bucket_name = f'bucket-test-39180-{int(time.time())}'
+        self.prov_obj.basic_io_write_read_validate(bucket_name=bucket_name, s3t_obj=self.s3t_obj)
```
Comment (sonalk0209) [inline tests/cft/test_io_workload.py:43]: so no need to assert in test in case it fails? or its taken care in lib itself?

Comment (priyankaborawake) [reply tests/cft/test_io_workload.py:43]: its taken care in lib itself.

**Review by ketanarlulkar — APPROVED** [review]

File: tests/cft/test_io_workload.py (hunk 1 above)
Comment (priyankaborawake) [reply tests/cft/test_io_workload.py:38]: added s3_data_path

File: tests/cft/test_io_workload.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    def setup_class(cls):
+        """Setup class."""
+        cls.prov_obj = ProvDeployK8sCortxLib()
+        cls.s3t_obj = S3TestLib()
+
+    @pytest.mark.lc
+    @pytest.mark.sanity
+    @pytest.mark.s3_data_path
+    @pytest.mark.tags("TEST-39180")
+    def test_39180(self):
```
Comment (sonalk0209) [inline tests/cft/test_io_workload.py:41]: can we rather say test_basic_io???

**Review by sonalk0209 — APPROVED** [review]
@priyankaborawake one minor comment

**Pull request merged**